.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
```

Comment:
It's running your `gulp` tasks for you: https://github.com/mozilla/node-firefox-find-simulators/blob/master/gulpfile.js ^_^

https://github.com/mozilla/node-firefox-build-tools/blob/master/lib/gulp-tasks.js